spec/plans/provision.fmf

```diff
@@ -44,6 +44,11 @@ example: |
         directly on the localhost. Note that for some actions like
         installing additional packages you need root permission or
         enabled sudo.
+
+        Commands ``tmt-abort``, ``tmt-file-submit`` and ``tmt-report-result``
+        can be used only if created manually or if tmt is installed from the rpm.
```

Comment:
I would not document the `created manully` workflow.
But if you insist, we should provide some snippet, how should one do that, so it is more clear.
I would avoid this, feels like a workaround.